makefile

```diff
@@ -239,11 +239,20 @@ $(buildDir)/make-tarball:cmd/make-tarball/make-tarball.go
 	@GOOS="" GOARCH="" $(gobin) build -o $@ $<
 	@echo $(gobin) build -o $@ $<
 
+$(buildDir)/sign-executable:cmd/sign-executable/sign-executable.go
+	@mkdir -p $(buildDir)
+	@GOOS="" GOARCH="" $(gobin) build -o $@ $<
```

Comment:
Removed.
I'm confused why it would be necessary to clear it for `build-cross-compile` more than any other of the cmd scripts that are running on the native platform. I removed it for `build-cross-compile` too, but lmk if I'm missing something.